go/vt/tabletserver/logstats.go

```diff
@@ -50,7 +50,9 @@ type LogStats struct {
 	Error                *TabletError
 }
 
-func newLogStats(methodName string, ctx context.Context) *LogStats {
+// NewLogStats constructs a new LogStats with supplied Method and ctx field values, and the StartTime field set to
+// the present time.
+func NewLogStats(methodName string, ctx context.Context) *LogStats {
```

Comment:
During commit, the `golint` git hook throws a warning that the `context.Contex` argument should come first.  I chose to just acknowledge the warning and not make further changes.